sdk/attestation/azure-security-attestation/README.md

```diff
@@ -0,0 +1,180 @@
+
+There are four major components in the preview SDK: 
+- [SGX and TPM enclave attestation.](#attestation)
+- [MAA Attestation Token signing certificate discovery and validation.](#attestation-token-signing-certificate-discovery-and-validation)  
+- [Attestation Policy management.](#policy-management)
+- [Attestation policy management certificate management](#policy-management-certificate-management) (yes, policy management management).
+
+Each attestation instance operates in one of two separate modes of operation:
+* AAD mode.
+* Isolated mode
+
+In "AAD" mode, access to the service is controlled solely by Azure Role Based Access Control.  In "Isolated" mode, 
+the client is expected to provide additional evidence to prove that the client is authorized
+to modify the service.
+
+Finally, each region in which the Microsoft Azure Attestation service is available supports a "shared" instance, which
+can be used to attest SGX enclaves which only need verification against the azure baseline (there are no policies applied to the shared instance). TPM attestation is not available in the shared instance.
+While the shared instance requires AAD authentication, it does not have any RBAC policies - any customer with a valid AAD bearer token can attest using the shared instance. 
+
+
+### Attestation
+SGX or TPM attestation is the process of validating evidence collected from 
+a trusted execution environment to ensure that it meets both the Azure baseline for that environment and customer defined policies applied to that environment.
+
+### Attestation token signing certificate discovery and validation
+Most responses from the MAA service are expressed in the form of a JSON Web Token. This token will be signed by a signing certificate
+issued by the MAA service for the specified instance. If the MAA service instance is running in a region where the service runs in an SGX enclave, then
+the certificate issued by the server can be verified using the [oe_verify_attestation_certificate() API](https://openenclave.github.io/openenclave/api/enclave_8h_a3b75c5638360adca181a0d945b45ad86.html). 
+
+### Policy Management
+Each attestation service instance has a policy applied to it which defines additional criteria which the customer has defined.
+
+For more information on attestation policies, see [Attestation Policy](https://docs.microsoft.com/azure/attestation/author-sign-policy)
+
+### Policy Management certificate management.
+When an attestation instance is running in "Isolated" mode, the customer who created the instance will have provided
+a policy management certificate at the time the instance is created. All policy modification operations require that the customer sign
+the policy data with one of the existing policy management certificates. The Policy Management Certificate Management APIs enable 
+clients to "roll" the policy management certificates.
+
+
```

Comment:
"How to create various clients using builder and using endpoint" : Most of our SDK have this before we show exmaple code which uses the client.